docs/release_notes_pdk.md

```diff
@@ -2,6 +2,18 @@
 
 New features, enhancements, and resolved issues for the PDK 1.x release series.
 
+## PDK 1.18.1
+
+### Resolved issues
+
+#### Ensure template have access to metadata during update/convert
+During an `update` or `convert` it was possible for the module metadata config
+to be empty, due to the templates not having access. [PDK-1653](https://tickets.puppetlabs.com/browse/PDK-1653)
```

Comment:
```suggestion
Because templates didn't have access to metadata, it was possible for the module metadata config
to be empty during an `update` or `convert`. [PDK-1653](https://tickets.puppetlabs.com/browse/PDK-1653)
```